src/client/render/cameras.js

```diff
@@ -343,31 +343,58 @@ function TrackingCamera(
 	let _self = this;
 
 	// Some default camera values. Could be added as an option later?
-	let _XZOffset = 3;
-	let _YOffset = 7;
-	let _minZoom = .2;
-	let _maxZoom = 3;
+	const _XZOffset = 3;
+	const _YOffset = 7;
+	const _minZoom = .2;
+	const _maxZoom = 3;
 
 	/**
 	 * Enables the controls
 	 */
 	this.enable = function() {
-		let func;
-
 		this.enabled = true;
 
 		this.update = function() {
 			update.bind(this)();
 		};
 
 		// Using the scrolling wheel allows a user to zoom the camera (closer or further away from the marble that is being tracked)
-		func = function(event) {
-			let newMultiplier = _self.distanceMultiplier + .05 * event.deltaY;
-			if (_self.enabled === true && newMultiplier > _minZoom && newMultiplier < _maxZoom) {
-				_self.distanceMultiplier = newMultiplier;
+		// 1 pixel should result in a .002 zoom change. When `deltaMode` is not 0, we assume notched intervals, of 16px each.
+		const zoomWheelFunction = function(event) {
+			if (_self.enabled === true) {
+				_self.distanceMultiplier = Math.min(_maxZoom, Math.max(_minZoom,
+					_self.distanceMultiplier + (
+						event.deltaY
+						* (event.deltaMode !== 0 ? 0.032 : .002)
```

Comment:
I'm not really fond of these magic numbers. Could we add these to the `const`s above? Just like the camera movement speed, I'd imagine we may want to make this configurable at some point.